multi-agent/lib/mas/session/building/workflow_session_factory.py

```
@@ -36,11 +36,13 @@ def __init__(
             engine_name: str,
             auth_service: Optional[AuthService] = None,
             platform_config: Optional[PlatformConfig] = None,
+            tracing_service: Optional[Any] = None,
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `tracing_service`

(ANN401)